index.html

```diff
@@ -772,8 +772,8 @@
 <!-- ---------------------------------------------------- -->
 
 <!-- Adding lottie player -->
-<script src="https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/5.12.2/lottie.min.js" integrity="sha512-jEnuDt6jfecCjthQAJ+ed0MTVA++5ZKmlUcmDGBv2vUI/REn6FuIdixLNnQT+vKusE2hhTk2is3cFvv5wA+Sgg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+<script src="js/lottie-player.js"></script>
+<script src="js/lottie.js" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```

Comment:
this change is not required. different lottie file versions keep getting released.
I had added this. Please revert this.